orchagent/aclorch.cpp

```diff
@@ -1341,7 +1342,7 @@ sai_status_t AclOrch::bindAclTable(sai_object_id_t table_oid, AclTable &aclTable
         auto& portAcls = m_portBind[portOid];
 
         sai_attribute_t attr;
-        attr.id = SAI_PORT_ATTR_INGRESS_ACL_LIST;
+        attr.id = SAI_PORT_ATTR_INGRESS_ACL;
```

Comment:
Port Ingress ACL is no longer a list of objects (list of tables), it is a single object, either a table or a group
So seems a group should be created and bounded for each port
The following 
    // Port OID to vector of ACL OIDs
    map <sai_object_id_t, vector<sai_object_id_t>> m_portBind;
Should be changed from vector to a single object